hypertrace-core-graphql-span-schema/src/test/java/org/hypertrace/core/graphql/span/dao/BaseDaoTest.java

```diff
@@ -0,0 +1,220 @@
+package org.hypertrace.core.graphql.span.dao;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Instant;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import lombok.Value;
+import lombok.experimental.Accessors;
+import org.hypertrace.core.graphql.attributes.AttributeModel;
+import org.hypertrace.core.graphql.attributes.AttributeModelMetricAggregationType;
+import org.hypertrace.core.graphql.attributes.AttributeModelType;
+import org.hypertrace.core.graphql.common.request.AttributeAssociation;
+import org.hypertrace.core.graphql.common.request.AttributeRequest;
+import org.hypertrace.core.graphql.common.request.ResultSetRequest;
+import org.hypertrace.core.graphql.common.schema.arguments.TimeRangeArgument;
+import org.hypertrace.core.graphql.common.schema.attributes.AttributeScope;
+import org.hypertrace.core.graphql.common.schema.results.arguments.filter.FilterArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.filter.FilterOperatorType;
+import org.hypertrace.core.graphql.common.schema.results.arguments.filter.FilterType;
+import org.hypertrace.core.graphql.common.schema.results.arguments.order.OrderArgument;
+import org.hypertrace.core.graphql.context.GraphQlRequestContext;
+import org.hypertrace.core.graphql.span.request.SpanRequest;
+import org.hypertrace.gateway.service.v1.common.ValueType;
+import org.hypertrace.gateway.service.v1.log.events.LogEvent;
+import org.hypertrace.gateway.service.v1.log.events.LogEventsResponse;
+import org.hypertrace.gateway.service.v1.span.SpanEvent;
+import org.hypertrace.gateway.service.v1.span.SpansResponse;
+
+class BaseDaoTest {
```

Comment:
nit - if these are just statics to reference, usually is better to use a static util rather than inheritance.